telegram/ext/jobqueue.py

```diff
@@ -172,6 +172,11 @@ def run_repeating(self, callback, interval, first=None, context=None, name=None)
             :class:`telegram.ext.Job`: The new ``Job`` instance that has been added to the job
             queue.
 
+        Notes:
```

Comment:
The notes in jobqueue are unrelated so please remove them from this PR

Reply:
done, sorry